### ISSUE TITLE
Readme

### ISSUE THREAD

**Issue by melissamcewen** [benjaffe/chrome-okc-plugin#128]
I like to write documentation and I have been hacking around on this for a bit, so I was wondering if you'd be interested in having me edit the readme and add some new sections? My proposal would be:
* Original goal - preserve a lot of your original text, give people the ideas behind it
* Description
** How it should work
* Categories
** Some information about the current categories available
* Customization
** How to add new categories
** How to add new questions to a category
* Build instructions

**Comment by benjaffe** [comment]
@melissamcewen Yes!! This sounds fantastic, and I'll gladly accept any documentation-related pull requests you send my way. :)

**Comment by benjaffe** [comment]
I'd also be happy to change the description text for the plugin in the chrome store -- if you want to, just  make a new file with that text and hack on it.

**Comment by instantgreen** [comment]
@melissamcewen 
This would be tremendously helpful since recently I wasn't able to add new categories via the usual way! Something seems to have changed.
